src/client.rs

```diff
@@ -393,6 +436,8 @@ pub enum ClientFlowEvent {
 pub enum ClientFlowError {
     #[error("Stream was closed")]
     StreamClosed,
+    #[error("Queue receiver was closed")]
+    QueueClosed(#[source] mpsc::error::SendError<()>),
```

Comment:
Where is this error emitted? Can't find it.

Reply:
It is in the hidden bloc (because diff too big), see https://github.com/duesee/imap-flow/pull/165/files#diff-b9990512dda829fb6d9ef34c240dce3fd5dfd967b089a973a0e54db5784cfe82R725.

Reply:
Ah, okay. Then it would be better to make it a variant of `SchedulerError` instead of `ClientFlowError`.